src/rosdistro/vcs.py

```diff
@@ -0,0 +1,80 @@
+# Software License Agreement (BSD License)
+#
+# Copyright (c) 2016, Clearpath Robotics
```

Comment:
Since some part of this code was refactored from another file I think it should keep the previous copyright statement (in addition to Clearpath).